pkg/api/v1beta2/types.go

```diff
@@ -1103,6 +1105,18 @@ type ResourceQuotaList struct {
 	Items []ResourceQuota `json:"items"`
 }
 
+// NFSMount represents an NFS mount that lasts the lifetime of a pod
+type NFSMount struct {
+	// Server is the hostname or ip address of the NFS server
+	Server string `json:"server"`
+
+	// SourcePath is the exported NFS share
+	SourcePath string `json:"sourcePath"`
+
+	// MountOptions currently enables read only mounts, valid value is "ro"
+	MountOptions string `json:"mountOptions"`
```

Comment:
is this supposed to be comma seperated?  Or should it start its life as a []string?